### ISSUE TITLE
Fix contrast issues

### ISSUE THREAD

**Issue by dhinrichs-scottlogic** [gsproston-scottlogic/prompt-injection#614]
## Description
some of our text input fields fail contrast ratio tests.

### Acceptance Criteria
- Given the input field is disabled
- When you check the colour contrast (e.g. Firefox right click "Inspect Accessibility Properties" select "Check for issues: Contrast")
- The disabled defence configuration inputs pass WCAG criteria

### Screenshots
![Image](https://github.com/ScottLogic/prompt-injection/assets/125262707/4630c6ba-b532-4edc-922a-db0567139287)
